src/resources/filters/normalize/pandoc3.lua

```diff
@@ -35,27 +34,21 @@ function parse_pandoc3_figures()
 end
 
 function render_pandoc3_figures() 
-  -- 2023-02-17: only do this for Jats output since the other pandoc 3.1 writers
-  -- still appear to accept "implicit figures".
-  if _quarto.format.isJatsOutput() then
-    return {
-      Para = function(para)
-        if (#para.content == 1 and para.content[1].t == "Image" and
-            hasFigureRef(para.content[1])) then
-          local img = para.content[1]
-          -- quarto.utils.dump(img.caption)
-          local caption = img.caption
-          return pandoc.Figure(
-            pandoc.Plain(para.content[1]),
-            {
-              short = nil,
-              long = {pandoc.Plain(caption)}
-            },
-            img.attr)
-        end
-      end,
-    }
-  else
-    return {}
-  end
+  return {
```

Comment:
It might be worth looking at how Pandoc distributes attributes across figures. When I was getting figures to work in the visual editor I noted that class and keyvalue attributes were left on the image whereas the identifier was kept on the `Figure`. I _think_ the below includes all of the attributes on both (not sure if that would have undesirable side effects?)

Reply:
Okay, I am good with this then!